src/morse/index.js

```diff
@@ -58,6 +58,26 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) {
+  if (text === '') {
+    return ''
+  }
+
+  try {
+    text = text.trim().replace(/\s{3,}/ig,'  ').split(' ')
```

Comment:
Interesting.

Reply:
@bondz, thanks for the feedback, really impactful.